src/Compilers/VisualBasic/Portable/Symbols/Source/OverrideHidingHelper.vb

```diff
@@ -90,7 +90,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
         Public Shared Function SignaturesMatch(sym1 As Symbol, sym2 As Symbol, <Out()> ByRef exactMatch As Boolean, <Out()> ByRef exactMatchIgnoringCustomModifiers As Boolean) As Boolean
             ' NOTE: we should NOT ignore extra required parameters as for overloading
             Const mismatchesForOverriding As SymbolComparisonResults =
-                (SymbolComparisonResults.AllMismatches And (Not SymbolComparisonResults.MismatchesForConflictingMethods)) Or SymbolComparisonResults.CustomModifierMismatch
+                (SymbolComparisonResults.AllMismatches And (Not SymbolComparisonResults.MismatchesForConflictingMethods)) Or SymbolComparisonResults.CustomModifierMismatch Or SymbolComparisonResults.TupleNamesMismatch
```

Comment:
This is probably un-necessary, since that bit in the results will be ignored below.